kubernetes/client/clientset/clientset.go

```diff
@@ -1,15 +1,19 @@
 package clientset
 
 import (
-	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset_generated/clientset/typed/compose/v1beta1"
+	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta1"
+	composev1beta2 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta2"
 	glog "github.com/golang/glog"
 	discovery "k8s.io/client-go/discovery"
 	rest "k8s.io/client-go/rest"
 	flowcontrol "k8s.io/client-go/util/flowcontrol"
 )
 
+// Interface defines the methods a compose kube client should have
+// FIXME(vdemeester) is it required ?
```

Comment:
Ah :sweat_smile: I need to find the answer for it :stuck_out_tongue:

Reply:
Yes please 😄